src/App.vue

```diff
@@ -415,34 +425,33 @@ export default class App extends Mixins(BcolMixin, DateMixin, FilingTemplateMixi
     }
   }
 
-  /** Set up any required fields/ validation if a Name request is present */
+  /** Fetches NR and validates it. */
   private async processNameRequest (filing: any): Promise<void> {
     try {
-      const nameRequest = filing.incorporationApplication.nameRequest
+      const nrNumber = filing.incorporationApplication.nameRequest.nrNumber
 
-      // ensure we have an NR number
-      if (!nameRequest.nrNumber) {
```

Comment:
Two reasons we don't need this: (1) In the case of a numbered business IA, we now have a nameRequest object without nrNumber. (2) When this method is called, we know we have a NR to fetch.

Note that this nameRequest object (ie, from draft filing) is **not** the name request data from namex/NRO. This object matches the name_request schema that the Legal API requires. See buildFiling() for how this object is saved. See generateNameRequestState() for how we load the store with NR data.